### ISSUE TITLE
File logger ignores LOG_FORMAT option

### ISSUE THREAD

**Issue by vykulakov** [alerta/alerta#1051]
**Issue Summary**

Now I'm experimenting with Alerts logs and want to write them to a file. But when I see logs in the file, they are always in the JSON format. I've tried to change the LOG_FORMAT option but logs in the file always stay the same - looks like the file logger ignores this option.

**Environment**
- OS: Linux
- API version: 7.3.2
- Deployment: self-hosted
- WSGI environment: nginx/gunicorn

- Database: Postgres

- Server config:
  Auth enabled? Yes
  Auth provider? GitLab

- web UI version: 7.3.2
- CLI version: 7.3.0

**To Reproduce**
Steps to reproduce the behaviour:
1. Add/change the following Alerta Server options:
```
LOG_HANDLERS = ['file', 'console']
LOG_FILE = '/tmp/alerta.log'
LOG_FORMAT = 'simple'
```
2. Restart Alerta Server.
3. Compare logs from the console and within the file.
4. Find that logs in the file are in the JSON format.
5. Try any other LOG_FORMAT and find that the format of logs in the file stays the same.

**Expected behaviour**
Logs within the file have the format that specified with the LOG_FORMAT option.

**Screenshots**
Not required

**Additional context**
Not required

NOTE: Please provide as much information about your issue as possible.
Failure to provide basic details about your specific environment make
it impossible to know if an issue has already been fixed, can delay a
response and may result in your issue being closed without a resolution.

**Comment by vykulakov** [comment]
Oh, I found this commit: https://github.com/alerta/alerta/commit/22d1119bcfa7e92af9b08b84f4b68212ad807031

**Comment by vykulakov** [comment]
Duplicates: #1037
Already fixed: #1038
